src/inventory.gql

```diff
@@ -121,7 +121,7 @@ type Mutation {
 """The input of a product item batch creation"""
 input CreateProductItemBatchInput {
   """The product variant id of the product item"""
-  productVariantId: UUID!
```

Comment:
Why rename these inputs?
The schema all other services use includes the Id suffix for input ids

Reply:
agreed for consistency with other services